src/pages/[platform]/reference/library-releases/index.mdx

```diff
@@ -0,0 +1,60 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Amplify libraries version & support',
+  description: 'Reference for Amplify libraries version & support',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'vue'
+  ]
+};
+
+export async function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export async function getStaticProps() {
+  return {
+    props: {
+      meta,
+    }
+  };
+}
+
+<InlineFilter filters={['javascript', "angular", "nextjs", "react", "vue"]}>
```

Comment:
we should be safe to remove this filter in favor of using the platform filter in the page meta

Reply:
Removed.